### ISSUE TITLE
RoleMixin as a role of a Category

### ISSUE THREAD

**Issue by jmeinlschmidt** [OntoUML/OntoUML#66]
There is an example (see image 1) of using RoleMixin as a subtype of Category. However, this options is not specified in Allowed supertypes property (see image 2). If provided example is an incorrect, is there any possibility for Categories to have a Role specified?

https://ontouml.readthedocs.io/en/latest/classes/nonsortals/rolemixin/index.html

![image](https://user-images.githubusercontent.com/13226284/142613893-5df31e69-04f5-4d0c-aa24-8540047b61e4.png)

![image](https://user-images.githubusercontent.com/13226284/142614052-1f8f899d-c4a3-4f16-9d02-f38d02eb8c13.png)

**Comment by pedropaulofb** [comment]
@jmeinlschmidt, you are right. RoleMixins specializing categories are even an identified pattern, see the [RoleMixin Alternative pattern](https://ontouml.readthedocs.io/en/latest/patterns/rolemixin_alt/index.html). I am going to fix this issue. Thank you for reporting it!
